src/test/java/com/hillel/items_exchange/controller/UserControllerTest.java

```diff
@@ -264,18 +272,34 @@ void updateChild_NotValidDto_ShouldThrowIllegalIdentifierException() throws Exce
         assertTrue(mvcResult.getResponse().getContentAsString().contains("Not all children from dto present in"));
     }
 
-    private String getJsonOfChildrenDto(long maleId, long femaleId, int year) {
-        return asJsonString(List.of(
-                getChildDto(maleId, LocalDate.of(year, 3, 3), "male"),
-                getChildDto(femaleId, LocalDate.of(year, 4, 4), "female")
-        ));
+    @Test
+    @WithMockUser(username = "admin")
+    @DataSet("database_init.yml")
+    void addChild_BadAmount_ShouldThrowConstraintViolationException() throws Exception {
+        final MvcResult mvcResult = mockMvc.perform(post("/user/child")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(badAmountChildDtoJson)
+                .accept(MediaType.APPLICATION_JSON))
+                .andDo(print())
+                .andExpect(status().isBadRequest())
+                .andReturn();
+        assertTrue(Objects.requireNonNull(mvcResult.getResolvedException()).getMessage()
```

Comment:
Instead of using ``Objects.requireNonNul`` you could use ``assertNotNull()`` but it isn't critical here, just FYI.

Reply:
If you do not mind, I will leave it as it was. I think now it looks more concise